orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/PipelineLauncher.java

```diff
@@ -61,7 +61,6 @@ public PipelineLauncher(ObjectMapper objectMapper,
       .withExecutingInstance(currentInstanceId)
       .withExecutionEngine(Execution.V2_EXECUTION_ENGINE)
       .withNotifications((List<Map<String, Object>>) config.get("notifications"))
-      .withId()
```

Comment:
@duftler this was the line that was breaking the indexing in redis. Didn't even appear to be necessary for anything.